sourcegraph/about
# Update 'More than search' pdf & add to related resources

Closes #5611. Updates 'More than search' pdf & adds it to **Related Resources** section on select pages.

### Changelog
""

### Test
1. Ensure prettier has standardized the proposed changes.
2. Ensure **Related Resources** are updated accordingly in changes files.
